tests/test_models/test_seg_heads/test_panoptic_fpn_head.py

```diff
@@ -14,8 +14,8 @@ def test_init_weights(self):
         head = PanopticFPNHead(
             num_things_classes=2,
             num_stuff_classes=2,
-            in_channels=1,
```

Comment:
consider use a smaller number to reduce GPU memory cost

Reply:
Just initialized, not really running.